iron-query-params.html

```diff
@@ -56,8 +56,13 @@
       if (this._dontReact) {
         return;
       }
-      this.paramsString = this._encodeParams(this.paramsObject)
-          .replace(/%3F/g, '?').replace(/%2F/g, '/').replace(/'/g, '%27');
+      let _encParams = this._encodeParams(this.paramsObject)
```

Comment:
Oops, @fooloomanzoo I messed up. I meant:
```
if (this._dontReact || !this.paramsString)
```

You might be right, let me think more about it.